uwblueprint/jump-math
# Omar/setup chakra

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Setup-Chakra-UI-2423ca05523d4a4baeda5cc1328aeb34)



## Implementation description
* Setup chakra UI and added the appropriate dependencies



## Steps to test
1. run docker-compose up
2. go to localhost:3000 and make sure the webpage loads without errors



## What should reviewers focus on?
* 


## Checklist
